src/TLuaInterpreterMapper.cpp

```diff
@@ -3699,6 +3688,7 @@ int TLuaInterpreter::setRoomWeight(lua_State* L)
 
     pR->setWeight(w);
     host.mpMap->setUnsaved(__func__);
+    host.mpMap->update();
```

Comment:
Changing the weight of the room does not produce any visible changes - so this is not needed.

Reply:
See https://github.com/Mudlet/Mudlet/pull/7187#discussion_r1544529665